packages/cli/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "peerDependencies": {},
   "devDependencies": {
+    "spectacle": "workspace:spectacle@*",
```

Comment:
If it's a runtime it will be included in the entire bin when we build this cli as runnable. We likely need to do a script to extract out the version number from the other package.json rather than some kind of cross-package dependency. @gksander and I were pondering this, but I'm really mixed if typing the version is easier. We do that on one-page today.

Reply:
Yeah I'm mixed too ... hard-coding it seems easy, but not as easy to maintain.

Reply:
I think we do need to change this before publishing, because currently trying to run the CLI (outside this repo) I think will fail.

Reply:
Yup! the CLI is not ready to be published yet. We still need to build the interactive prompts. Just flags are working.